### PR TITLE
post: cfp launch announcement

### PR DESCRIPTION
**Pull request by DennyPerez18**
# PyLadies Pull Request
_Please document the following. We will aim to be as timely as possible in processing your pull request, however as this is a volunteer staffed project anticipate at least a week for pull requests to start to be reviewed._ 

## Is this pull requesting adding or updating a [PyLadies chapter website](https://github.com/pyladies/pyladies#to-add-a-new-pyladies-location)? 
 

- [ ] Yes
- [X] No

## Have you confirmed the website builds locally? See [here](https://github.com/pyladies/pyladies#run-the-site-locally).

- [ ] Yes
- [X] No

##  Does your pull request pass the `netlify/pyladies/deploy-preview` check?

- [ ] Yes
- [ ] No

# Pull Request Checklist

- [X] Fill out all sections above
- [ ] Share pull request in PyLadies Slack channel #project-tech-infra

/cc @pyladies/tech-and-infra-admins

---

I wasn't able to deploy it locally to test if it works because I was having the following issue:

```bash
>> Searching for configuration file
@@ Deprecated configuration file found
..  rename config.yml to mynt.yml
@@ Configuration setting `domain` missing protocol
..  defaulting to `https`
!! Renderer could not be imported: jinja
```
Maybe someone could help me with that later 😇

### CODE REVIEW

File: www/_posts/2023/2023-07-28-cfp-launch-announcement.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+**🌟 Share Your Knowledge: Call for Papers**
+
+We invite speakers from all backgrounds and experiences to share their insights, expertise, and stories with our vibrant community. Whether you're a seasoned Python developer, an industry expert, or a first-time speaker, we encourage you to submit your proposals!
+
+As we are a global community, you are welcome to present in any one of the following languages: \*\*English, Spanish, Portuguese, French, and Chinese.\*\*. However, due to the limitation of our proposal review team, we can only review talk proposals in the following languages: “English, Spanish, Portuguese, and French”. Please indicate in your proposal which language you will present in, if it is different than the language of the proposal.
+
+**🔗 Submit Your Proposal:**
+
+Please click here to submit your proposal. \[https://pretalx.com/orga/event/pyladiescon-2023/live\]
```
[the remark this answers is not in the document]
Comment (jtemporal) [reply www/_posts/2023/2023-07-28-cfp-launch-announcement.md:27]: with that fixed I believe this can be merged no problem ;)

**Comment by jtemporal** [comment]
Another note: I don't seem to be able to see the event on pretalx with the link in the post 🤔

File: www/_posts/2023/2023-07-28-cfp-launch-announcement.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**🎉 What is PyLadiesCon?**
+
+PyLadiesCon is an extraordinary event designed to promote diversity, learning, and empowerment within the Python community. It celebrates women in Python, aiming to create a welcoming and inclusive environment for all attendees.
+
+**🌟 Share Your Knowledge: Call for Papers**
+
+We invite speakers from all backgrounds and experiences to share their insights, expertise, and stories with our vibrant community. Whether you're a seasoned Python developer, an industry expert, or a first-time speaker, we encourage you to submit your proposals!
+
+As we are a global community, you are welcome to present in any one of the following languages: \*\*English, Spanish, Portuguese, French, and Chinese.\*\*. However, due to the limitation of our proposal review team, we can only review talk proposals in the following languages: “English, Spanish, Portuguese, and French”. Please indicate in your proposal which language you will present in, if it is different than the language of the proposal.
```
Comment (Mariatta) [inline www/_posts/2023/2023-07-28-cfp-launch-announcement.md:23]:
```suggestion
As we are a global community, we welcome talks in the following languages: **English, Español, Português, Français, and 中文**. 
```

File: www/_posts/2023/2023-07-28-cfp-launch-announcement.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+**🔗 Submit Your Proposal:**
+
+Please click here to submit your proposal. \[https://pretalx.com/orga/event/pyladiescon-2023/live\]
+
+The topics can range from technical talks to best practices, case studies, and more. Every story and perspective is valuable, and your contribution will enrich the conference experience for everyone.
+
+**🗓️ Important Dates:**
+
+Call for Papers Launch: July 27th
```
Comment (Mariatta) [inline www/_posts/2023/2023-07-28-cfp-launch-announcement.md:33]:
```suggestion
Call for Papers Launch: August 4th
```

File: www/_posts/2023/2023-07-28-cfp-launch-announcement.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+**🌟 Share Your Knowledge: Call for Papers**
+
+We invite speakers from all backgrounds and experiences to share their insights, expertise, and stories with our vibrant community. Whether you're a seasoned Python developer, an industry expert, or a first-time speaker, we encourage you to submit your proposals!
+
+As we are a global community, we welcome talks in the following languages: **English, Español, Português, Français, and 中文**. 
+
+**🔗 Submit Your Proposal:**
+
+[Please click here to submit your proposal.](https://pretalx.com/orga/event/pyladiescon-2023/live)
```
Comment (Mariatta) [inline www/_posts/2023/2023-07-28-cfp-launch-announcement.md:27]:
```suggestion
[Please click here to submit your proposal.](https://pretalx.com/pyladiescon-2023/cfp)
```

**Review by Mariatta — APPROVED** [review]

**Pull request merged**